sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReceiveLinkProcessor.java

```diff
@@ -218,12 +222,15 @@ public void onNext(ServiceBusReceiveLink next) {
             currentLink = next;
             next.setEmptyCreditListener(() -> 0);
 
+            unreceivedMessages.set(0);
+
             currentLinkSubscriptions = Disposables.composite(
-                next.receive().publishOn(Schedulers.boundedElastic()).subscribe(
```

Comment:
Remove `publishOn(Schedulers.boundedElastic())` because`ServiceBusReactorReceiver#receive()` already use publishOn(). [code](https://github.com/Azure/azure-sdk-for-java/blob/6d827ec4869bb797a858eaed855290d5926a080a/sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorReceiver.java#L149-L153)